h2o-algos/src/main/java/hex/gam/MatrixFrameUtils/GamUtils.java

```diff
@@ -93,6 +92,12 @@ public static GLMParameters copyGAMParams2GLMParams(GAMParameters parms, Frame t
     setParamField(parms, glmParam, true, field2);
     glmParam._train = trainData._key;
     glmParam._valid = valid==null?null:valid._key;
+    glmParam._nfolds = 0; // always set nfolds to 0 to disable cv in GLM.  It is done in GAM
```

Comment:
Can be removed. This is always zero by default and the class is instantiated by using the default constructor.

Reply:
@pavel:  In this case, I am copying GAM parameters to GLM parameters.  If I want cross-validation, I will set nfolds = 3, and therefore after the copying of parameters, GLM parameter nfolds will be 3.  Hence, I need to set it to 0.